base/sparse/cholmod.jl

```diff
@@ -418,6 +418,13 @@ function copy_dense{Tv<:VTypes}(A::Dense{Tv})
     d
 end
 
+function sort!{Tv<:VTypes}(S::Sparse{Tv})
```

Comment:
this doesn't do the same thing as julia `sort!` it shouldn't be called that in julia code. you're strictly confusing people by matching the C names of a library with a horribly messy APi